packages/ui/src/components/TimelineAdmin.tsx

```diff
@@ -0,0 +1,217 @@
+import { twMerge } from 'tailwind-merge'
+import { faCalendar, faPauseCircle } from '@fortawesome/free-regular-svg-icons'
+import { faCheck, faPause, faSync } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+// import { STATUS } from '@gbl-uzh/platform/dist/lib/util'
+import SegmentEntry from './SegmentEntry'
+
+// interface Props {
+//   periodStatus: string
+//   segmentStatus: string
+// }
+
+function TimelineAdmin() {
+  const periods = [
```

Comment:
this should still be refactored to be passed through props. just to remember, assuming the component is not yet done :)